tests/integration/test_tableclasses_vectorcassandratable.py

```diff
@@ -16,87 +16,106 @@
 N = 8
 
 
-@pytest.mark.usefixtures("db_session", "db_keyspace")
-class TestVectorCassandraTable:
-    def test_crud(self, db_session: Session, db_keyspace: str) -> None:
-        table_name = "v_ct"
-        db_session.execute(f"DROP TABLE IF EXISTS {db_keyspace}.{table_name};")
-        #
-        t = VectorCassandraTable(
-            session=db_session,
-            keyspace=db_keyspace,
-            table=table_name,
-            vector_dimension=2,
-            primary_key_type="TEXT",
+def crud(db_session: Session, db_keyspace: str) -> None:
```

Comment:
Any specific reason for taking these out of a test class? Just curious

Reply:
Just that the test class doesn’t add anything. So this is simpler.